src/components/styles/header.css

```diff
@@ -0,0 +1,38 @@
+/* stylelint-disable-next-line block-opening-brace-space-before */
```

Comment:
- [ ] Also, please consider removing stylelint-disabler as well. If you cannot remove it. You can use the above hint that I gave :)